scale/data/data/value.py

```diff
@@ -27,6 +28,15 @@ def __init__(self, name, param_type):
         self.name = name
         self.param_type = param_type
 
+    def copy(self):
```

Comment:
Not sure I understand the point of this method? Is there going to be more logic added later?